apps/openmw/mwdialogue/dialoguemanagerimp.cpp

```diff
@@ -647,6 +647,9 @@ namespace MWDialogue
             if (!info->mSound.empty())
                 sndMgr->say(actor, info->mSound);
         }
+
+        if (!info->mResultScript.empty())
```

Comment:
Info is not checked for NULL, which is causing crashes on my end. Please fix it.